src/main/java/com/google/jenkins/plugins/storage/AbstractUpload.java

```diff
@@ -231,61 +258,57 @@ public boolean forResult(Result result) {
   }
 
   /**
-   * The bucket name specified by the user, which potentially contains unresolved symbols, such as
-   * $JOB_NAME and $BUILD_NUMBER.
+   * @return The bucket name specified by the user, which potentially contains unresolved symbols,
+   *     such as $JOB_NAME and $BUILD_NUMBER.
    */
   public String getBucket() {
     return bucketNameWithVars;
   }
 
-  /** NOTE: old name kept for deserialization */
-  private final String bucketNameWithVars;
-
-  /** Whether to surface the file being uploaded to anyone with the link. */
+  /** @param sharedPublicly Whether to surface the file being uploaded to anyone with the link. */
   @DataBoundSetter
   public void setSharedPublicly(boolean sharedPublicly) {
     this.sharedPublicly = sharedPublicly;
   }
 
+  /** @return Whether to surface the file being uploaded to anyone with the link. */
   public boolean isSharedPublicly() {
     return sharedPublicly;
   }
 
-  private boolean sharedPublicly;
-
-  /** Whether to attempt the upload, even if the job failed. */
+  /** @param forFailedJobs Whether to attempt the upload, even if the job failed. */
   @DataBoundSetter
   public void setForFailedJobs(boolean forFailedJobs) {
     this.forFailedJobs = forFailedJobs;
   }
 
+  /** @return Whether to attempt the upload, even if the job failed. */
   public boolean isForFailedJobs() {
     return forFailedJobs;
   }
 
-  private boolean forFailedJobs;
-
   /**
-   * Whether to indicate in metadata that the file should be viewable inline in web browsers, rather
-   * than requiring it to be downloaded first.
+   * @param showInline Set whether to indicate in metadata that the file should be viewable inline
+   *     in web browsers, rather than requiring it to be downloaded first.
    */
   @DataBoundSetter
   public void setShowInline(boolean showInline) {
     this.showInline = showInline;
   }
 
+  /**
+   * @return Whether to indicate in metadata that the file should be viewable inline in web
+   *     browsers, rather than requiring it to be downloaded first.
+   */
   public boolean isShowInline() {
     return showInline;
   }
 
-  private boolean showInline;
-
   /**
-   * The path prefix that will be stripped from uploaded files. May be null if no path prefix needs
-   * to be stripped.
-   *
-   * <p>Filenames that do not start with this prefix will not be modified. Trailing slash is
-   * automatically added if it is missing.
+   * @param pathPrefix The path prefix that will be stripped from uploaded files. May be null if no
+   *     path prefix needs to be stripped.
+   *     <p>Filenames that do not start with this prefix will not be modified. Trailing slash is
```

Comment:
The unclosed paragraph tag confused me but I saw that it's supported from this [post](https://stackoverflow.com/questions/33787753/javadoc-closing-tags). I don't like that the maven format plugin forces it to be on the same line. What do you think about removing it in favor of an extra newline?

Reply:
As discussed, when we run mvn verify, the formatter seems to put the <p> tag after we remove it and put the comment on a new line, so we will keep it.